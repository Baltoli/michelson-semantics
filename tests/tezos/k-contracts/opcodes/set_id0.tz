$contract { parameter (set string);
storage (set string);
code { CAR ; NIL operation ; PAIR }
 } ;
$param { "a" ; "b" ; "c" } ;
$storage {} ;
$ExpectedOutput { "a" ; "b" ; "c" } ;

