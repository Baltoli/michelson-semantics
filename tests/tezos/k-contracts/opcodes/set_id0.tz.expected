<generatedTop>
  <k>
    .
  </k>
  <stack>
    Pair .List SetItem ( "a" )
    SetItem ( "b" )
    SetItem ( "c" )
  </stack>
  <paramtype>
    set .AnnotationList string .AnnotationList
  </paramtype>
  <storagetype>
    set .AnnotationList string .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
